CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,7 @@
-- Fix: avoid raise mongo alarm when a measure is not maching a group configuration
-- Upgrade NodeJS version from 10 to 12 in Dockerfile due to Node 10 End-of-Life
-- Set Nodejs 12 as minimum version in packages.json (effectively removing Nodev10 from supported versions)
-- Add list of environment variables which can be protected by Docker Secrets
+- Fix: avoid raise mongo alarm when a measure is not maching a group configuration
+- Fix: Set 60 seconds for default mqtt keepalive option (#413)
+- Upgrade NodeJS version from 10 to 12 in Dockerfile due to Node 10 End-of-Life
+- Set Nodejs 12 as minimum version in packages.json (effectively removing Nodev10 from supported versions)
+- Add: Support of multimeasure for MQTT and AMQP transport (#462)
+- Add: virtual host configuration added to AMQP transport (config.amqp.vhost and IOTA_AMQP_VHOST env var)
+- Add list of environment variables which can be protected by Docker Secrets
```

Comment:
Not sure if the conflict in CHANGES_NEXT_RELEASE has been correctly solved... CHANGES_NEXT_RELEASE should contain the current entries in master (i.e. https://github.com/telefonicaid/iotagent-json/blob/master/CHANGES_NEXT_RELEASE) plus the line associated to this PR (i.e. ` - Add: virtual host configuration added to AMQP transport (config.amqp.vhost and IOTA_AMQP_VHOST env var)`).